package.json

```diff
@@ -5,10 +5,10 @@
     "ws": "~0.4.28"
   },
   "dependencies": {
-    "google-closure-compiler": "20200224.0.0",
+    "acorn": "7.3.1",
+    "google-closure-compiler": "20200920.0.0",
```

Comment:
Oh hah, I didn't realise it's that simple, thought you're using a separate JAR file somewhere. Thanks!